cdi-core/src/test/java/com/linkedin/cdi/util/HttpRequestMethodTest.java

```diff
@@ -67,6 +69,54 @@ public void setUp() {
     headers = new HashMap<>();
   }
 
+  /**
+   * Test HttpGet_xe method with parameters
+   * Note: This works when the URI template has all the variable parameters.
+   * If there are any additional parameters, then URIBuilder encodes all parameters while building.
+   *
+   * @throws UnsupportedEncodingException
+   */
+  /*
+  @Test
```

Comment:
Why are we commenting out this test?

Reply:
This fix fixes the current problem of not encoding the url params. 
But, not for the case that I have added in the comments. 
Since, I had that test already in place. Decided to keep it commented out.